<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpandToNeighborsConfig</class>
 <widget class="QWidget" name="ExpandToNeighborsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="radioButton_all">
     <property name="text">
      <string>All</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_sur">
     <property name="text">
      <string>Only surface</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_sur_angle">
     <property name="text">
      <string>Only surface with angle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget__dihedral" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max. Dihedral Angle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_dihedral">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pi radians</string>
        </property>
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>PI Rads.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_sur_angle</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget__dihedral</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
